Maya: Add skeletonMesh and skeletonAnim SET docs (#341)
ynput/ayon-documentation: website/docs/addon_maya_artist.md

When you've prepared your hierarchy, it's time to create *Rig instance* in AYON.
Select your whole rig hierarchy and go **AYON → Create...**. Select **Rig**.
A set named after the rig product is created in your scene to mark rig parts for export. Notice that it has
four children object sets - `controls_SET`, `out_SET`,  `skeletonMesh_SET` and `skeletonAnim_SET`. Put your controls into `controls_SET`
and geometry to `out_SET`. You should end up with something like this:

![Maya - Rig Hierarchy Example](assets/maya/artist/rig_hierarchy_example.jpg)

The other two products `skeletonMesh_SET` and `skeletonAnim_SET` are used for the fbx workflow.

:::tip Enable FBX for animation product type
Users need to enable **Collect Fbx Animation** before publishing it as animation product.
![Maya - Collect Fbx Animation](assets/maya/artist/animation_collect_fbx_animation.png)

:::

> The `skeletonMesh_SET` defines the static FBX skeleton and mesh, put any skinned geometry here usually the same used for 'out_SET' too.

Reviewer: ```suggestion
The `skeletonMesh_SET` defines the static FBX skeleton mesh, put any skin binded geometry here - usually the same ones used for 'out_SET'.
```

> ![Maya - Collect Fbx Animation](assets/maya/artist/animation_collect_fbx_animation.png)

Reviewer 2: IMO, we shouldn't put settings in artist docs. as this is related to admin.
instead, we can just mention that admins can enable or disable it.

> The `skeletonAnim_SET` defines the skeleton consisting of bones hierarchy used for the animation product (FBX). Put your rig's skeleton hierarchy driving the animation here.

Reviewer: ```suggestion
The `skeletonAnim_SET` defines the skeleton consisting of bones hierarchy used for the animation product (FBX). Put your bones hierarchy driving the animation here.
```

When you've prepared your hierarchy, it's time to create *Rig instance* in AYON.
Select your whole rig hierarchy and go **AYON → Create...**. Select **Rig**.
Set is created in your scene to mark rig parts for export. Notice that it has

> four products - `controls_SET`, `out_SET`,  `skeletonMesh_SET` and `skeletonAnim_SET`. Put your controls into `controls_SET`

Reviewer 3: ```suggestion
four children object sets - `controls_SET`, `out_SET`,  `skeletonMesh_SET` and `skeletonAnim_SET`. Put your controls into `controls_SET`
```
Best to avoid confusion here with AYON products?

and geometry to `out_SET`. You should end up with something like this:

![Maya - Rig Hierarchy Example](assets/maya/artist/rig_hierarchy_example.jpg)

> The other two products `skeletonMesh_SET` and `skeletonAnim_SET` are proposed for fbx workflow.

Reviewer 3: ```suggestion
The other two products `skeletonMesh_SET` and `skeletonAnim_SET` are used for the fbx workflow.
```

Reviewer 3: Probably even better to create a dedicated subsection for this.
```suggestion
#### FBX skeleton and mesh

The other two products `skeletonMesh_SET` and `skeletonAnim_SET` are used for the fbx workflow.
```

The `skeletonMesh_SET` defines the static FBX skeleton and mesh, as an FBX product directly from the rig publish,
you should put your skinned character geometry with skeleton in the set.

The `skeletonAnim_SET` defines the animated content that should be published to FBX from an animation scene

> for the animation product type. You should include your animated skeleton data merely into the set.

Reviewer 3: ```suggestion
for the animation product type. This set should include only your animated skeleton.
```

Also, just wanted to state - I am still confused that `skeletonMesh` needs BOTH the mesh and the skeleton band this set only includes the skeleton. Why does the mesh one need both? It might be worth it if the docs would clarify the reasoning.

> Users need to enable **Collect Fbx Animation** before publishing it as animation product.

Reviewer 3: This is best added into a bigger TIP or NOTE block.

```suggestion

:::tip Enable FBX for animation product type
Users need to enable **Collect Fbx Animation** before publishing it as animation product.
:::
```

> ![Maya - Rig Hierarchy Example](assets/maya/artist/rig_skeletonAnim_skeletonMesh_set.png)

Reviewer 3: Personally I find this screenshot quite confusing.

- What if there are multiple meshes? Do they need to be grouped and the group put in the `skeletonMesh_SET` or must it contain direct mesh members only?
- The skeletonAnim set looks very confusing because visually in the screenshot it doesn't contain skeletons at all? It might be worth changing it around so that in the screenshot it's clear that _THAT_ is what is the skeleton and maybe hide whatever is the "Guides" there?

Reviewer 2: Additionally, I think having one screenshot would be better.
Or maybe putting both example screenshots near each other in a table.

When you've prepared your hierarchy, it's time to create *Rig instance* in AYON.
Select your whole rig hierarchy and go **AYON → Create...**. Select **Rig**.

> Set is created in your scene to mark rig parts for export. Notice that it has

Reviewer 2: This line looks unclear to me. 
Maybe something like this look clearer?
```suggestion
A set named after the rig product is created in your scene to mark rig parts for export. Notice that it has
```

four children object sets - `controls_SET`, `out_SET`,  `skeletonMesh_SET` and `skeletonAnim_SET`. Put your controls into `controls_SET`
and geometry to `out_SET`. You should end up with something like this:

![Maya - Rig Hierarchy Example](assets/maya/artist/rig_hierarchy_example.jpg)

The other two products `skeletonMesh_SET` and `skeletonAnim_SET` are used for the fbx workflow.

> The `skeletonMesh_SET` defines the static FBX skeleton and mesh, as an FBX product directly from the rig publish,
> you should put your skinned character geometry with skeleton in the set.

Reviewer: ```suggestion
The `skeletonMesh_SET` defines the static FBX skeleton and mesh, put any skinned geometry here usually the same used for 'out_SET' too.
```

> The `skeletonAnim_SET` defines the animated content that should be published to FBX from an animation scene
> for the animation product type. This set should include only your animated skeleton.

Reviewer: ```suggestion
The `skeletonAnim_SET` defines the skeleton consisting of bones hierarchy used for the animation product (FBX). Put your rig's skeleton hierarchy driving the animation here.
```

:::tip Enable FBX for animation product type
Users need to enable **Collect Fbx Animation** before publishing it as animation product.

> ![Maya - Collect Fbx Animation](assets/maya/artist/animation_collect_fbx_animation.png)

Reviewer 2: I think having the collector only in the screenshot is better. 

![image](https://github.com/user-attachments/assets/d8180635-c108-43ae-827a-87f00d0110ae)
